source/patterns/02-organisms/content-header/content-header~ar.yaml

```diff
@@ -3,3 +3,8 @@ attributes:
   lang: ar
 contentTitle:
   text: عنوان المحتوى
+downloadIconLink:
+  attributes:
+    dir: rtl
```

Comment:
Not needed?

Reply:
It is needed, unfortunately.

Reply:
Hmm, this doesn't look right.

There's a lack of support for `float: inline-x` (only Firefox), so doesn't work in Chrome. Probably need to add fallback that uses `dir()` for that case?

Though looking at it, the individual pattern doesn't have the right `html@dir` (it gets set on the pattern, so it's like changing direction). Should probably resolve this.

Reply:
So individual pattern pages are easy to fix, it's the listings which are a problem as they share the same `<html>`. Would need https://github.com/libero/libero/issues/88.

Reply:
Wonder if we could somehow get a fallback to the pattern root as well as `<html>`? 🤔

Reply:
For now we can supply the directionality as necessary to get the individual patterns, and lists of patterns, displaying in Pattern Lab, and make sure the `inset()` mixin doesn't output the CSS `inset` shorthand property. If no other solution suddenly reveals itself, this slightly increases the priority of https://github.com/libero/libero/issues/88.